core/src/main/java/com/opensymphony/xwork2/interceptor/ChainingInterceptor.java

```diff
@@ -160,7 +160,20 @@ private void copyStack(ActionInvocation invocation, CompoundRoot root) {
         Map<String, Object> ctxMap = invocation.getInvocationContext().getContextMap();
         for (Object object : list) {
             if (shouldCopy(object)) {
-                reflectionProvider.copy(object, invocation.getAction(), ctxMap, prepareExcludes(), includes);
+                String actionConfigClassName = invocation.getProxy().getConfig().getClassName();
```

Comment:
What `editableClass` means here? Shouldn't it be a `originClass` or `targetClass`?